packages/headless/src/database/mongodb/store-admin/store-admin.model.ts

```diff
@@ -9,29 +9,67 @@ const StoreAdminSchema = new Schema<StoreAdmin>(
       default: () => randomUUID(),
       unique: true,
     },
-    info: {
-      name: String,
-      email: {
-        type: String,
-        unique: true,
-      },
-      phone: String,
+    firstName: {
+      type: String,
+      trim: true,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      trim: true,
+      required: true,
     },
-    password: String,
     role: {
-      name: String,
-      roleId: String,
+      name: {
+        type: String,
+        trim: true,
+        required: true,
+      },
+      roleId: {
+        type: String,
+        trim: true,
+        required: true,
+      },
       roleType: {
         type: String,
-        enum: ['OWNER', 'BRANCH_MANAGER'],
+        trim: true,
+        required: true,
       },
     },
-    // branchId
-    branch: {
+    storeId: {
+      type: String,
+      trim: true,
+      index: true,
+      required: true,
+    },
+    branchId: {
```

Comment:
use plural name for multiple data storing

Reply:
Done